parachains/overview.md

```diff
@@ -1,57 +1,55 @@
+- **[XCM (Cross Consensus Messaging)](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/xcm/index.html){target=\_blank}**: The messaging format for communicating with other parachains and the relay chain.
+- **[Polkadot](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/polkadot/index.html){target=\_blank}**: The relay chain that provides security and coordination.
 
-### Substrate
+### Substrate: The Foundation
 
-Substrate is a Software Development Kit (SDK) that uses Rust-based libraries and tools to enable you to build application-specific blockchains from modular and extensible components. Application-specific blockchains built with Substrate can run as standalone services or in parallel with other chains to take advantage of the shared security provided by the Polkadot ecosystem. Substrate includes default implementations of the core components of the blockchain infrastructure to allow you to focus on the application logic.
+Substrate provides the core infrastructure that every parachain is built upon. It handles the low-level blockchain functionality, allowing you to focus on your application's unique features. Substrate includes implementations for networking, database management, consensus participation, and the execution environment for your runtime.
 
-Every blockchain platform relies on a decentralized network of computers—called nodes—that communicate with each other about transactions and blocks. In general, a node in this context is the software running on the connected devices rather than the physical or virtual machine in the network. As software, Substrate-based nodes consist of two main parts with separate responsibilities:
+Every Polkadot SDK node consists of two main components:
 
-- **Client**: Services to handle network and blockchain infrastructure activity.
+- **Client (Host)**: Handles infrastructure services
 
-    - Native binary.
-    - Executes the Wasm runtime.
-    - Manages components like database, networking, mempool, consensus, and others.
-    - Also known as "Host".
+    - Native binary that runs on validator and collator nodes
+    - Executes the Wasm-compiled runtime
+    - Manages networking, database, mempool, and block production
+    - Interfaces with the relay chain for validation
 
-- **Runtime**: Business logic for state transitions.
+- **Runtime (State Transition Function)**: Contains your business logic
 
-    - Application logic.
-    - Compiled to [Wasm](https://webassembly.org/){target=\_blank}.
-    - Stored as a part of the chain state.
-    - Also known as State Transition Function (STF).
+    - Defines how your Polkadot SDK node processes transactions
+    - Compiled to [Wasm](https://webassembly.org/){target=\_blank} for deterministic execution
+    - Stored on-chain and upgradeable via governance
```

Comment:
```suggestion
    - Defines how your Polkadot SDK node processes transactions.
    - Compiled to [Wasm](https://webassembly.org/){target=\_blank} for deterministic execution.
    - Stored on-chain and upgradeable via governance.
```

Just a reminder, you can reference the Google developer documentation style guide for when to add punctuation to lists: https://developers.google.com/style/lists?hl=en#capitalization-and-end-punctuation

Most of these items have verbs, so for consistency, we should apply punctuation to all items